src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx

```diff
@@ -43,7 +43,7 @@ export const WalletManagementButtons: React.FC<
   return (
     <>
       <div ref={walletManagementButtonsRef}>
-        {redirectToLearn ? (
+        {redirectToApp ? (
```

Comment:
Not really fan of that, can you do it directly into this component instead of passing it through 2 components please?

Reply:
Veryy good idea! Thanks